src/Dropzone/DefaultContent.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Icon from '../Icon';
+import { FileUpload } from '../../icons';
+
+const DefaultContent = () => (
+  <>
+    <div>
+      <Icon src={FileUpload} style={{ height: '48px', width: '48px' }} />
+    </div>
+    <p>
+      Drag and drop your file here or click to upload.
```

Comment:
I think we can generalise that, at least I managed to do it to error messages 😄 

Here it gets trickier though with all the translations, e.g you could specify only `maxSize` or `minSize` or both and different messages should get rendered.. and accept is also tricky, what should we write if you specify 3 extensions? "Upload PNG, JPG, or JPEG files" I guess, but for translations that's totally different from "Upload PNG or JPG files" *I think*.

Anyway, I'd like to treat this as a small follow up ticket if that's okay (if this one gets merged this week that is, otherwise I'll try to update this PR)